### ISSUE TITLE
Problem expanding macro 'getter' since Crystal v0.34.0 release

### ISSUE THREAD

**Issue by Val** [ysbaddaden/minitest.cr#39]
Given `shard.yml` file:
```yaml
name: test_minitest.cr
version: 0.1.0
dependencies:
    minitest:
      github: ysbaddaden/minitest.cr
      branch: master
```
And `test_minitest.cr` file:
```crystal
class Foo
  def bar
    "baz"
  end
end

require "minitest/autorun"

class FooTest < Minitest::Test
  def foo
    @foo ||= Foo.new
  end

  def test_that_foo_bar_baz
    assert_equal "baz", foo.bar
  end
end

describe Foo do
  let(:foo) { Foo.new }

  describe "when asked about bar" do
    it "must respond baz" do
      foo.bar.must_equal("baz")
    end
  end
end
```
Since [Crystal v0.34.0 release](https://crystal-lang.org/2020/04/06/crystal-0.34.0-released.html) I get a build error on expanding macro `getter`:
```shell
$ crystal run test_minitest.cr 
Showing last frame. Use --error-trace for full trace.

There was a problem expanding macro 'getter'

Code in /usr/share/crystal/src/file/error.cr:5:3

 5 | getter file : String
     ^
Called macro defined in macro 'macro_139691680479536'

 117 | macro getter(*names, &block)

Which expanded to:

 > 2 |         
 > 3 |           
 > 4 |             @file : String
                   ^
Error: instance variable '@file' of Exception, with File::Error < Exception, is already declared as (String | Nil) (trying to re-declare as String)
```

**Comment by ysbaddaden** [comment]
Fixed in the just released v0.5.1!

**Comment by Val** [comment]
Thank you!
